src/core/unit.c

```diff
@@ -2411,6 +2411,10 @@ static int unit_log_resources(Unit *u) {
                         return log_oom();
                 iovec[n_iovec++] = IOVEC_MAKE_STRING(TAKE_PTR(t));
 
+                /* If value is 0, we don't log it in the MESSAGE= field. */
+                if (v == 0)
```

Comment:
Not a matter, but maybe better to rename `v` -> `value` for consistency with the below?
Feel free to ignore this comment.